File: Simulation.R
set.seed(31415926)

N = 10**6

M = 0

for (i in 0:N){
  V = rnorm(1, 0, 1)
  Y = 2
  Z = V + Y + rnorm(1, 0, 0.2)
  if (Z >= 0 && Z <= 1){
    M = M + 1
  }
}

M/N*100

File: Simulation.py
import numpy as np

np.random.seed(31425926)
N = 10**6

V = np.random.poisson(5, N)
X = V + np.random.poisson(3, N)
Y = 3 * np.ones(N, dtype=int)
Z = X + Y + np.random.poisson(1, N)

Wsk = len(np.where(Z == 10)[0]) / N

print(Wsk*100)